homeassistant/components/harmony/select.py

```diff
@@ -52,7 +52,7 @@ def options(self) -> list[str]:
         return [ACTIVITY_POWER_OFF] + sorted(self._data.activity_names)
 
     @property
-    def current_option(self):
+    def current_option(self) -> str | None:
```

Comment:
Can this actually be `None`? I added it, but maybe this would be better:
```suggestion
    def current_option(self) -> str:
```

Reply:
the `current_activity` can return `None`: https://github.com/ehendrix23/aioharmony/blob/8eb4cd02c75a96a23feb6bb7decc3f2bf49eb6e5/aioharmony/harmonyclient.py#L983, so it's better to keep the `None` here I think